### PR TITLE
remove second CA used for kubelet auth in favor of webhook auth

### PR DESCRIPTION
**Pull request by mikedanese**
partial fixes upgrade test.

### CODE REVIEW

**Comment by liggitt** [comment]
the apiserver only supports client cert auth to kubelets today

**Comment by mikedanese** [comment]
@liggitt I think that's fine. I'm building a client cert for the apiserver using the original CA in the change. I'm attempting to duplicate what we do in kubeadm.

**Comment by liggitt** [comment]
> @liggitt I think that's fine. I'm building a client cert for the apiserver using the original CA in the change. I'm attempting to duplicate what we do in kubeadm.

ah, "code review by reading the PR title" fails me again

**Comment by wojtek-t** [comment]
@mikedanese - any progress on that?

**Comment by mikedanese** [comment]
@wojtek-t this needs to be reviewed.

**Comment by mikedanese** [comment]
@k8s-bot cross build this

File: cluster/common.sh [hunk 1]
```diff
@@ -999,6 +996,7 @@ function generate-certs {
     # this puts the cert into pki/ca.crt and the key into pki/private/ca.key
     ./easyrsa --batch "--req-cn=${PRIMARY_CN}@$(date +%s)" build-ca nopass
     ./easyrsa --subject-alt-name="${SANS}" build-server-full "${MASTER_NAME}" nopass
+    ./easyrsa build-client-full kube-apiserver nopass
```
Comment (cjcullen) [inline cluster/common.sh]: Does this run on upgrades?

**Comment by cjcullen** [comment]
/lgtm

Talked w/ @mikedanese about the overall plan and this is good to at least unblock upgrade testing.

**Comment by wojtek-t** [comment]
Thanks a lot @mikedanese and @cjcullen !

**Pull request merged**